src/schedule-and-details/ScheduleAndDetails.scss

```diff
@@ -0,0 +1,37 @@
+@import "./basic-section/BasicSection";
+@import "./credit-section/CreditSection";
+@import "./pacing-section/PacingSection";
+
+.schedule-header-title {
+  @extend %heading-h2;
+
+  margin-bottom: 1.75rem;
+
+  .schedule-header-title-subtitle {
+    @extend %body-small;
+
+    display: block;
+    color: $text-color-base;
+  }
+}
+
+.section-container:not(:last-child) {
```

Comment:
Is this selector a child or parent of other selectors in this file? If so, please make them nested.